### PR TITLE
Add Vida Caffe

### LINKED ISSUE #9148: Vida e Caffè (JS Blob)
### Brand name

Vida e Caffè

Coffee shop franchise in South Africa

### Wikidata ID

Q7927650
https://www.wikidata.org/wiki/Q7927650
https://www.wikidata.org/wiki/Special:EntityData/Q7927650.json

### Store finder url(s)
https://vidaecaffe.com/contact/stores/

Official Url(s): https://vidaecaffe.com/
- [x] I have run the below and if anything in generated, added it.
- [x] I have manually checked for a storefinder
- [ ] This is not a manufacturer listing outlets operating under their own brands / this will not create duplicates
`pipenv run scrapy links https://vidaecaffe.com/`
`pipenv run scrapy sitemap --pages https://vidaecaffe.com/`
`pipenv run scrapy sf --brand-wikidata=Q7927650 https://vidaecaffe.com/`
```
```
Official Url(s): http://www.vidaecaffe.co.za/
- [ ] I have run the below and if anything in generated, added it.
- [ ] I have manually checked for a storefinder
- [ ] This is not a manufacturer listing outlets operating under their own brands / this will not create duplicates
`pipenv run scrapy links http://www.vidaecaffe.co.za/`
`pipenv run scrapy sitemap --pages http://www.vidaecaffe.co.za/`
`pipenv run scrapy sf --brand-wikidata=Q7927650 http://www.vidaecaffe.co.za/`
```
```

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fixes #9148

### CODE REVIEW

File: locations/spiders/vida_caffe_za.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            for time in location["Times"]:
+                item["opening_hours"].add_ranges_from_string(time["label"] + " " + time["times"])
+
+        # Features
+        # 'store_features': ['free-wifi', 'halaal', 'mobile-payment', 'redeem-vitality-rewards'
+        apply_yes_no(Extras.WIFI, item, "free-wifi" in location["store_features"])
+        apply_yes_no(Extras.HALAL, item, "halaal" in location["store_features"])
+
+        # Mobile payment?
+        # apply_yes_no(PaymentMethods.CONTACTLESS, item, "mobile-payment" in location["store_features"])
```
Comment (CloCkWeRX) [inline locations/spiders/vida_caffe_za.py:50]: @arrival-spring do you have any local knowledge around how this is mapped? I am assuming it isn't paywave, but something like https://en.wikipedia.org/wiki/M-Pesa ?

Comment (arrival-spring) [reply locations/spiders/vida_caffe_za.py:50]: I don't have any local knowledge, but from the website and app listing it looks to me like "payment:app"

**Comment by arrival-spring** [comment]
Also, note this is not ZA only, but locations outside of ZA are currently getting the wrong country code

File: locations/spiders/vida_caffe_za.py (hunk 1 above)
Comment (CloCkWeRX) [inline locations/spiders/vida_caffe_za.py:50]:
```suggestion
        apply_yes_no(PaymentMethods.APP, item, "mobile-payment" in location["store_features"])
```

**Pull request merged**